react-youtube-adaptive-loading/src/components/Debugging/CheckboxWithLabel/CheckboxWithLabel.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
The purpose of this debugging option was to make it trivial for developers to toggle the lite or full mode without needing to trip into one of the hardware/network conditions required for experiencing one or the other. We are also adding a query-parameter for making this switch to enable simple benchmarking use-cases.